python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -55,10 +55,9 @@ class MultiPostProcessor(IPostProcessor):
         'process_objects': ComponentItem('postprocessor', defvalue=[], hidden=1, doc='A list of Processors to run', sequence=1)
     })
 
-    def __init__(self):
+    def __init__(self, *args):
         super(MultiPostProcessor, self).__init__()
 
-    def __construct__(self, args):
         if len(args) == 1 or len(args) > 1 and isType(args, (GangaList, list)):
```

Comment:
Looking at this in more detail I don't see how this method can now be called as a user shouldn't be constructing a `MultiPostProcessor` from the GPI. I don't know if we do in object loading at all?
I'm going to clean it up to always expect a tuple and leave the rest of the code for now.